pandas/tests/test_categorical.py

```diff
@@ -306,20 +306,19 @@ def f():
         assert len(cat.codes) == 1
         assert cat.codes[0] == 0
 
-        # Catch old style constructor useage: two arrays, codes + categories
-        # We can only catch two cases:
+        # Catches - now disabled - for old style constructor useage:
```

Comment:
you don't need the 'Catches - now disabled', this is not longer relevant to a reader of the code